README.md

```diff
@@ -15,6 +15,31 @@ To access docfx 3 preview, install `docfx` using the latest version of [.NET Cor
 dotnet tool install -g docfx --version 3.0.0-* --add-source https://www.myget.org/F/docfx-v3/api/v2
 ```
 
+## Usage
```

Comment:
What about add a note before usage "The outputs are still some JSON files, not an HTML site, as the template system is still under design", to avoid letting down the community user who tries this...